int64 scenario_id
---
ScenarioInput[] inputs
